deployments/hadoop-yarn/ansible/34-setup-shuffler.yml

```diff
@@ -0,0 +1,38 @@
+#
+# <meta:header>
+#   <meta:licence>
+#     Copyright (c) 2021, ROE (http://www.roe.ac.uk/)
+#
+#     This information is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation, either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     This information is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#
+#     You should have received a copy of the GNU General Public License
+#     along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#   </meta:licence>
+# </meta:header>
+#
+#
+
+- name: "Copy Shuffle jar to tmp directory"
+  hosts: masters
+  tasks:
+    - fetch:
+        src: /opt/spark/yarn/spark-2.4.7-yarn-shuffle.jar
+        dest: /tmp/yarnspark-2.4.7-yarn-shuffle.jar
```

Comment:
This hard codes the Spark version into this Ansible script.

If the [spname](https://github.com/wfau/aglais/blob/10a106c6493cf208e7c01899683c2b20e301297a/deployments/hadoop-yarn/ansible/config/cclake-medium-04.yml#L43) setting for this deployment changes, then  this Ansible script will look for the wrong jar file.

By design different deployments can have different values for [spname](https://github.com/wfau/aglais/blob/10a106c6493cf208e7c01899683c2b20e301297a/deployments/hadoop-yarn/ansible/config/cclake-medium-04.yml#L43).